core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -86,6 +91,23 @@ public Long getDeadlineNanoTime() {
     return deadlineNanoTime;
   }
 
+  /**
+   * Retruns the Codec used for compression and decompression, or {@code null} if none is set.
+   */
+  @Nullable
+  public Codec getCompression() {
```

Comment:
getCompressionCodec seems like a decent name.  Codec is a COder / DECoder, but I have been using it as COmpressor / DECompressor.  Since this would be on the public API I tried to pick a name that a user would think of ("I want to turn on compression, let me look for methods with the word compression in them").  getCompressionCodec would fit that.

My one qualm is that it would be like an ATM Machine: slightly redundant in its name.